modules/modal/react/lib/ModalContent.tsx

```diff
@@ -172,7 +173,28 @@ const ModalContent = ({
     }
   };
 
-  return (
+  React.useEffect(() => {
+    const siblings = [...((document.body.children as any) as HTMLElement[])].filter(
```

Comment:
I forgot to make this line and https://github.com/Workday/canvas-kit/pull/419/files#diff-3fe25d4d6af6314872e682fe13b7f7ccR212 use a configurable container instead of just `document.body`. Applications may need a different container than `document.body`